Dockerfile.dapper

```diff
@@ -4,7 +4,8 @@ FROM ubuntu:16.04
 ARG DAPPER_HOST_ARCH
 ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH} DAPPER_ENV=REPO DAPPER_ENV=TAG \
     DAPPER_SOURCE=/go/src/github.com/rancher/submariner DAPPER_DOCKER_SOCKET=true \
-    TRASH_CACHE=${DAPPER_SOURCE}/.trash-cache HOME=${DAPPER_SOURCE} LINT_VERSION=v1.16.0 DAPPER_OUTPUT=output
+    TRASH_CACHE=${DAPPER_SOURCE}/.trash-cache HOME=${DAPPER_SOURCE} DAPPER_OUTPUT=output \
+    LINT_VERSION=v1.17.1 HELM_VERSION=v2.14.1 KIND_VERSION=v0.3.0 KUBEFED_VERSION=0.1.0-rc2
```

Comment:
Could you split these up into separate lines, like #82? It means version bumps produce nice diffs in future...

Reply:
Done with the latest commit.